src/pages/CategoriesPage/Table/DeleteCategoryColumn/DeleteCategoryColumnComponent.tsx

```diff
@@ -0,0 +1,25 @@
+import { Delete } from "@mui/icons-material";
+import { Button } from "@mui/material";
+import DeleteCategoryContainer from "../../DeleteCategory/DeleteCategoryContainer";
+import { styles } from "../../styles";
+
+interface Props {
+    id: number
+    onToggleDeleteStatusClick: () => void
+}
+
+const DeleteCategoryColumnComponent = (props: Props) => {
+    return (
+        <div>
```

Comment:
Do not use DIV. Use Box instead.

Reply:
All divs in the modal windows have been replaced by the Box components